### PR TITLE
fix: add retry when problem network limit order

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- fix `location = useLocation`
- hide limit order by chain support
- fix issues when network error=> retry max 2 times

### CODE REVIEW

File: src/components/swapv2/LimitOrder/useBaseTradeInfo.ts [hunk 1]
```diff
@@ -2,57 +2,88 @@ import { Currency, CurrencyAmount, Price, TokenAmount } from '@kyberswap/ks-sdk-
...
-    async (url: string) => {
-      if (!currencyOut || !currencyIn || !url) return
+      : null
+  }
+  const fetchData = async (url: string | null): Promise<BaseTradeInfo | undefined> => {
+    if (!currencyOut || !currencyIn || !url) return
+    const { data } = await axios.get(url, {
+      headers: {
+        'X-Request-Id': sentryRequestId,
+        'Accept-Version': 'Latest',
```
Comment (nguyenhuudungz) [inline src/components/swapv2/LimitOrder/useBaseTradeInfo.ts:44]: I think this isnt supported?

**Review by nguyenhuudungz — CHANGES_REQUESTED** [review]

File: src/components/swapv2/LimitOrder/useBaseTradeInfo.ts (hunk 1 above)
Comment (nguyenhoaidanh) [reply src/components/swapv2/LimitOrder/useBaseTradeInfo.ts:44]: I see kyberswap currently uses it in API `route/encode` (aggregator.ts bestTradeExactIn function)

Comment (nguyenhuudungz) [reply src/components/swapv2/LimitOrder/useBaseTradeInfo.ts:44]: It is safe to remove.

Comment (nguyenhuudungz) [reply src/components/swapv2/LimitOrder/useBaseTradeInfo.ts:44]: We do not use `Accept-Version` anywhere.

Comment (nguyenhoaidanh) [reply src/components/swapv2/LimitOrder/useBaseTradeInfo.ts:44]: this is still api route/encode, I think the backend need it for tracking, ...
if you confirm it is not necessary. I will remove.

**Review by nguyenhuudungz — APPROVED** [review]

**Pull request merged**